charts/opentelemetry-collector/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.47.0
+version: 0.48.0
```

```suggestion
version: 0.47.1
```

For this feature we only need to bump the patch.  Once you've updated the chart you'll need to re-run `make generate-examples CHARTS=opentelemetry-collector`